bip-0339.mediawiki

```diff
@@ -41,9 +41,10 @@ announcing and fetching transactions.
 
 # A new wtxidrelay message is added, which is defined as an empty message where pchCommand == "wtxidrelay".
```

Comment:
perhaps "A new wtxidrelay net message type is added"

Reply:
I'm going to leave this. My main motivation here was to update the BIP to be in line with the implementation for the wtxidrelay handshake and transaction fetching. It feels like scope creep to touch other parts of the document.